lib/buildResponses.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+var config = require('../config'),
+    constant = require('../constants'),
+    logger = require('logops');
+
+
+function invokeDashboard(request, options) { // Generic function for sending response configurations
+    request(options, function (err, response, body) {
+        if (err) {
+            logger.error('Request error %s', err);
+        } else {
+            if (response.statusCode = constant.HTTP_RESPONSE_OK) {
```

Comment:
do you know what are you doing here? ;) = "asigment"  == "comparation"

It's always "true"